app/src/main/java/io/left/reflect/RightMeshRecipientView.java

```diff
@@ -37,8 +36,39 @@
     // Keeps track of peers and populates the spinner.
     private MeshIdAdapter spinnerAdapter;
 
-    public RightMeshRecipientComponent() {
-        recipientId = null;
+    private RecipientChangedListener onRecipientChangedListener = null;
+
+    public RightMeshRecipientView(Context context) {
```

Comment:
A few of the methods in this class could use JavaDocs, even short ones. The interface at the bottom as well.

Reply:
Related: how hard would it be to have our checkstyle applied to this?

Reply:
@FrazerSeymour: Please check the next version for changes. I added JavaDocs for all methods and fixed warnings reported by Rightmesh checkstyle as well.
@compscidr It won't take long. @FrazerSeymour gave me a checkstyle plugin in Android Studio so setting checkstyle is much simpler now.